css-brochure-form/styles.css

```diff
@@ -0,0 +1,63 @@
+form {
+  width: 50%;
+  margin: auto;
+  margin-left: 6.8in;
+}
+
+p {
+  padding-left: 10px;
+  margin-bottom: 6px;
+  font-family: BrandonGrotesque, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+}
+
+input {
+  margin: 4px;
+  width: 53%;
+  padding: 10px;
+  border-style: solid;
+  border: 1px solid rgb(226 226 226);
+  border-radius: 5px;
+  font-size: 17px;
+  font-family: AtlasGrotesk, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  flex
```

Comment:
What is this? I hesitate to imagine that the pre-commit hooks would allow you to commit this. Make sure you only use `--no-verify` in situations that I have given approval for.